client/components/post-stats-card/index.tsx

```diff
@@ -1,10 +1,10 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Card, Button } from '@automattic/components';
+import { eye } from '@automattic/components/src/icons';
```

Comment:
Not perfect, but this is already happening in many places; we'd have to fix it everywhere.

Reply:
FWIW, the same icon is available in `@wordpress/icons` as `seen`.

Reply:
Also weird that the `Icon`s have a `gridicon` class for some reason, and are hidden with CSS 😔 That visibility should be controlled via prop if necessary.